law.cfg

```diff
@@ -104,6 +104,26 @@ cache_cleanup: $CF_WLCG_CACHE_CLEANUP
 cache_max_size: 50GB
 
 
+[wlcg_fs_desy_store_user]
+
+xrootd_base: root://dcache-cms-xrootd.desy.de:1094/pnfs/desy.de/cms/tier2/store/user/$CF_CERN_USER/$CF_STORE_NAME
+gsiftp_base: gsiftp://dcache-door-cms04.desy.de:2811/pnfs/desy.de/cms/tier2/store/user/$CF_CERN_USER/$CF_STORE_NAME
+base: &::gsiftp_base
```

Comment:
Where is this actually used? Did you mean to use this for the `wlcg_file_systems`?

Reply:
Yes, it's just a preparation for the case where we ask the students to store the outputs to their dCache space. Then they just need to add the `cf.TASK wlcg, wlcg_fs_desy_store_user` entry to the `[outputs]` section of the `law.cfg`.

Reply:
OK fine by me :+1: